docs/reference/release-notes/highlights-7.9.0.asciidoc

```diff
@@ -0,0 +1,13 @@
+[[release-highlights-7.9.0]]
+== 7.9.0 release highlights
+++++
+<titleabbrev>7.9.0</titleabbrev>
+++++
+
+coming[7.9.0]
+//NOTE: The notable-highlights tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+// tag::notable-highlights[]
+
+// end::notable-highlights[]
```

Comment:
As this is empty, I don't think we need to add it. I believe the `release-notes/highlights.asciidoc` is intended to now take the place of this file.

@debadair may have more context.

Reply:
Removed, if necessary it can be added in a later commit.